timereport/chalicelib/config.yaml

```diff
@@ -0,0 +1 @@
+valid_reasons: 
  - vab
  - sjuk
  - intern
  - semester
valid_actions:
  - add
  - list
  - show
log_level: DEBUG
backend_url: https://ywdi37qne9.execute-api.eu-north-1.amazonaws.com/api
```

Comment:
Git shows carriage return here instead of line feed.
We use line feed everywhere else so please use that instead